### PR TITLE
Fix bug in ONNXRT adapter where _rename_node function fails with model size > 2 GB

### PR DESCRIPTION
**Pull request by yuwenzho**
## Type of Change

bug fix
API not change

## Description

Fix bug in ONNXRT adapter where _rename_node function fails with model size > 2 GB

## Expected Behavior & Potential Risk

_rename_node function in ONNXRT adapter can handle ONNX models with size > 2 GB

## How has this PR been tested?

CI test

## Dependency Change?

no

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]

File: neural_compressor/adaptor/onnxrt.py [hunk 1]
```diff
@@ -760,8 +763,16 @@ def _rename_node(self, model):
             for idx, node in enumerate(model.graph.node):
                 if node_names.count(node.name) > 1:
                     node.name = node.op_type + '_nc_rename_' + str(idx)
-            onnx.save(model, os.path.join(self.work_space, "renamed_model.onnx")) 
-        return model
+            if model_wrapper.is_large_model:
+                onnx.save(model,
+                          os.path.join(self.work_space, "renamed_model.onnx"),
```
[the remark this answers is not in the document]
Comment (yuwenzho) [reply neural_compressor/adaptor/onnxrt.py:768]: The `_rename_model()` function here is a general method used to rename nodes with the same name, after which the model is saved as `renamed_model.onnx`. This is not specifically associated with situations dealing with > 2GB models. Logger will notify the user that they can check the `renamed_model.onnx` in workspace_path for newly generated node name.

**Review by mengniwang95 — APPROVED** [review]

**Pull request merged**